plugin/build.gradle.kts

```diff
@@ -21,6 +22,12 @@ repositories {
     jcenter()
 }
 
+tasks.withType<KotlinCompile>() {
+    kotlinOptions {
+        apiVersion = "1.3"
```

Comment:
Hrm... Gradle is still pretty heavily reliant upon version 1.2. Correct? If we start using 1.3 features, we'll break backwards compatibility with older versions of Gradle, won't we?

Reply:
apiVersion `1.3` was default target for the code compiled with Kotlin version `1.3.*`. Also Gradle itself is using Kotlin `1.3.*` for kotlin scripts, so I assume `1.3` here is correct.

Also samples works correctly.

Reply:
Oh, I was thinking about Kotlin 1.4
Nevermind. Thanks!